rskj-core/src/test/java/co/rsk/peg/federation/StandardMultisigFederationTest.java

```diff
@@ -85,10 +85,10 @@ void createInvalidFederation_aboveMaxScriptSigSize() {
         long creationBlockNumber = federation.getCreationBlockNumber();
         NetworkParameters networkParameters = federation.getBtcParams();
         FederationArgs federationArgs = new FederationArgs(newMembers, creationTime, creationBlockNumber, networkParameters);
-        ScriptCreationException exception =
-            assertThrows(ScriptCreationException.class,
-                () -> FederationFactory.buildStandardMultiSigFederation(federationArgs));
-        assertEquals(ABOVE_MAX_SCRIPT_ELEMENT_SIZE, exception.getReason());
+        // this is intentional since the serialization of federations just serializes the standard part
+        // when adding segwit, this thrown an exception since the p2sh-p2wsh erp federation with 20 members
+        // exceeded the redeem script size limit. So the decision was to remove the size limit check.
+        assertDoesNotThrow(() -> FederationFactory.buildStandardMultiSigFederation(federationArgs));
```

Comment:
I think this comment should go into the `des/serialization` method unit test instead. This border case should be tested for `co.rsk.peg.BridgeSerializationUtils#deserializeFederationAccordingToVersion`  method, passing 
a `STANDARD_MULTISIG_FEDERATION` and  a `P2SH_P2WSH_ERP_FEDERATION` federation with 20 members.

Besides that, what if `deserializeStandardMultisigFederationWithDeserializer` method do not assume that any fed can be an `standardMultiSig` and return just the fed args? By doing that, we don't need to remove the validation size.